lib/api/model/model.dart

```diff
@@ -249,10 +249,20 @@ class ZulipStream {
   final bool historyPublicToSubscribers;
   final int? messageRetentionDays;
 
-  final int streamPostPolicy; // TODO enum
-  // final bool isAnnouncementOnly; // deprecated; ignore
+  final StreamPostPolicy streamPostPolicy;
```

Comment:
nit: s/JsonEnum/enum/ in commit message:

    api: Change streamPostPolicy to JsonEnum

I wouldn't say this field *is* now "a JsonEnum" — rather, it's an enum, specifically a Dart enum. And that enum type is annotated with `JsonEnum`… which isn't so much a name of a kind of enum, but rather is an annotation saying how to relate some enum to JSON.